interop/tapir/src/test/scala/caliban/interop/tapir/TestApi.scala

```diff
@@ -64,7 +64,6 @@ object TestApi extends GenericSchema[TestService with Uploads] {
       maxDepth(30) @@                 // query analyzer that limit query depth
       timeout(3 seconds) @@           // wrapper that fails slow queries
       printSlowQueries(500 millis) @@ // wrapper that logs slow queries
-      printErrors @@                  // wrapper that logs errors
```

Comment:
I removed this since the tests were too noisy printing out all the (expected) errors from the added tests